src/sentry/dynamic_sampling/tasks.py

```diff
@@ -200,8 +200,11 @@ def process_transaction_biases(project_transactions: ProjectTransactions) -> Non
         DSElement(id=id, count=count) for id, count in project_transactions["transaction_counts"]
     ]
 
-    sample_rate = quotas.get_blended_sample_rate(organization_id=org_id)
-
+    sample_rate = get_sliding_window_sample_rate(
+        org_id=org_id,
+        project_id=project_id,
+        error_sample_rate_fallback=quotas.get_blended_sample_rate(organization_id=org_id),
+    )
```

Comment:
For orgs that don't have the feature flag, this will fallback to the blended rate?

Reply:
Good catch, I didn't cover for this case, will do now!